microservices/Api.Pdp/Controllers/WeatherForecastController.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
```

Comment:
clearly we don't need this file but that's ok

Reply:
its good to leave that type of thing for a while in case you want to see how they wired the logging for example. Until our code works..its always worth making sure the sample code works.